strings/strings.ko-KR.json

```
@@ -2443,38 +2443,38 @@
   "tech_neutronium_housing_effect": "%0에서의 생활 공간 비용 감소",
   "tech_unification": "통일",
   "tech_unification_desc": "%0 통일",
-  "tech_unification_effect": "모든 국가 지배",
+  "tech_unification_effect": "전 세계를 당신의 통치 하에 통일할 계획을 세웁니다.",
```

```suggestion
  "tech_unification_effect": "전 세계를 우리 통치 하에 통일할 계획을 세웁니다.",
```
아래의 다른 you처럼 여기도 우리를 쓰는게 더 자연스러워 보입니다